R/geom-sf.R

```diff
@@ -200,10 +201,15 @@ sf_grob <- function(x, lineend = "butt", linejoin = "round", linemitre = 10,
   fill <- x$fill %||% defaults$fill[type_ind]
   fill <- alpha(fill, alpha)
   size <- x$size %||% defaults$size[type_ind]
-  point_size <- ifelse(is_collection, x$size %||% defaults$point_size[type_ind], size)
+  linewidth <- x$linewidth %||% defaults$linewidth[type_ind]
+  point_size <- ifelse(
+    is_collection,
+    x$size %||% defaults$point_size[type_ind],
+    ifelse(is_point, size, linewidth)
+  )
```

Comment:
I might be wrong, but I think this can be left as is. `point_size` is used only for calculating `fontsize`, so `size` is appropriate here, isn't it?

Reply:
No, this is necessary to avoid erroring

Reply:
I see.